packages/datetime/src/dateRangePicker.tsx

```diff
@@ -528,33 +525,33 @@ function getStateChange(value: DateRange,
             *   - set the right DayPicker to +1
             */
             if (DateUtils.areSameMonth(nextValueStart, nextValueEnd)) {
-                const potentialLeftDisplayEqualsNextValueStart =
-                    potentialLeftDisplay.isSame(nextValueStartMonthDisplay);
-                const potentialRightDisplayEqualsNextValueStart =
-                    potentialRightDisplay.isSame(nextValueStartMonthDisplay);
+                const potentialLeftEqualsNextValueStart =
+                    potentialLeftMonthAndYear.isSame(nextValueStartMonthAndYear);
+                const potentialRightEqualsNextValueStart =
+                    potentialRightMonthAndYear.isSame(nextValueStartMonthAndYear);
 
-                if (potentialLeftDisplayEqualsNextValueStart || potentialRightDisplayEqualsNextValueStart) {
+                if (potentialLeftEqualsNextValueStart || potentialRightEqualsNextValueStart) {
                     // do nothing
                 } else {
-                    potentialLeftDisplay = nextValueStartMonthDisplay;
-                    potentialRightDisplay = nextValueStartMonthDisplay.getNextMonth();
+                    potentialLeftMonthAndYear = nextValueStartMonthAndYear;
+                    potentialRightMonthAndYear = nextValueStartMonthAndYear.getNextMonth();
                 }
             /*
             * Different start and end date months, adjust display months.
             */
             } else {
-                if (!potentialLeftDisplay.isSame(nextValueStartMonthDisplay)) {
-                    potentialLeftDisplay = nextValueStartMonthDisplay;
+                if (!potentialLeftMonthAndYear.isSame(nextValueStartMonthAndYear)) {
+                    potentialLeftMonthAndYear = nextValueStartMonthAndYear;
                 }
-                if (!potentialRightDisplay.isSame(nextValueEndMonthDisplay)) {
-                    potentialRightDisplay = nextValueEndMonthDisplay;
+                if (!potentialRightMonthAndYear.isSame(nextValueEndMonthAndYear)) {
+                    potentialRightMonthAndYear = nextValueEndMonthAndYear;
                 }
             }
         }
 
         returnVal = {
-            leftDisplayMonth: potentialLeftDisplay,
-            rightDisplayMonth: potentialRightDisplay,
+            leftMonthAndYear: potentialLeftMonthAndYear,
+            rightMonthAndYear: potentialRightMonthAndYear,
```

Comment:
but actually renaming _all_ these variables makes this thing dang near impossible to read. what if all the variables called it a `View` instead of `MonthAndYear`? so much shorter and more digestible.